doc/index.rst

```diff
@@ -12,17 +14,44 @@ pyFFS documentation
 *pyFFS* is a library to compute Fourier Series.
```

Comment:
do you think we should make this more specific? something like:

> pyFFS is a library to efficiently compute and interpolate Fourier Series coefficients.

Reply:
The main goal is to bring FS-based tools to the general public, so I think
```
pyFFS is a collection of efficient algorithms to compute Fourier Series and related transforms.
```
fits the bill.

Reply:
What do you think? In any case we should elaborate on the contents. The initial line is too sparse.